integration/helpers/create-fixture.ts

```diff
@@ -42,7 +42,7 @@ export async function createFixture(init: FixtureInit) {
     let url = new URL(href, "test://test");
     let request = new Request(url.toString(), {
       ...init,
-      signal: new AbortController().signal,
+      signal: init?.signal || new AbortController().signal,
```

Comment:
Chatted about the requirement of a signal and we want to stick with it to mimic the browser behavior, but we should make our `web-std-io` repo provide a signal automatically if not provided, and then remove the useless signals from vercel/netlify/architect adapters.

Reply:
Removed the unnecessary ones and allowed passthrough from config where necessary.